controls/ssh_spec.rb

```diff
@@ -40,6 +39,16 @@
     it { should_not be_writable.by('group') }
     it { should_not be_writable.by('other') }
   end
+  case os[:family]
```

Comment:
I did as advised except for the `git commit --fixup=`. I guess it's too late now. The amended commit was somehow connected to this branch of the conversation and it would show up here, right?

Next time.